packages/documentation/src/stories/foundation/color/color.stories.mdx

```diff
@@ -0,0 +1,119 @@
+import { Meta, Source } from '@storybook/addon-docs';
+import '@swisspost/design-system-styles/basics.scss';
+import './color.styles.scss';
+import { cssBackground, scssBackground } from './color.snippets';
+import { ColorSwatch } from './color-swatch';
+
+<Meta
+  title="Foundations/Color"
+  parameters={{
+    previewTabs: {
+      canvas: {
+        hidden: true,
+      },
+    },
+  }}
+/>
+
+# Color
+
+You can find the [color usage documentation](#usage) right below.
+
+## Brand colors
+
+<section className="color-square__section">
+  {['yellow', 'black', 'white'].map(color => (
+    <ColorSwatch key={color} color={color} />
+  ))}
+</section>
+
+## Shades of gray
+
+<section className="color-square__section">
+  {['gray-10', 'gray-20', 'gray-40', 'gray-60', 'gray-80'].map(color => {
+    return (
+      <article key={color}>
+        <div className="color-square">
+          <div
+            className="color-square__inner"
+            style={{ backgroundColor: `var(--post-${color}` }}
+          ></div>
+          <div class="color-square__content">
+            <h3>{color}</h3>
+            <dl>
+              <dt>CSS</dt>
+              <dd>
+                <code>var(--post-{color})</code>
+              </dd>
+              <dt>Sass</dt>
+              <dd>
+                <code>post.${color}</code>
+              </dd>
+            </dl>
+          </div>
+        </div>
+      </article>
+    );
+  })}
+</section>
+
+## Backgrounds
+
+<section className="color-square__section">
+  {['light', 'gray'].map(color => {
+    return <ColorSwatch key={color} color={color} />;
+  })}
+</section>
+
+## Signal colors
+
+<section className="color-square__section">
+  {['success', 'danger', 'warning', 'info'].map(color => {
+    return <ColorSwatch key={color} color={color} />;
+  })}
+</section>
+
+## Dark accent colors
+
+<section className="color-square__section">
+  {['nightblue', 'petrol', 'coral', 'olive', 'purple', 'aubergine'].map(color => {
+    return <ColorSwatch key={color} color={color} />;
+  })}
+</section>
+
+## Bright accent colors
+
+<section className="color-square__section">
+  {[
+    'nightblue-bright',
+    'petrol-bright',
+    'coral-bright',
+    'olive-bright',
+    'purple-bright',
+    'aubergine-bright',
+  ].map(color => {
+    return <ColorSwatch key={color} color={color} />;
+  })}
+</section>
+
+# Usage
+
+## CSS
+
+Import the Design System Styles package and choose from one of the provided style files:
+
+- **basics.css**: Just the basic styling for resets, typography and utility classes
+- **index.css**: The complete styling for internet facing applications
+- **intranet.css**: The complete styling for internal applications
+
+Use one of the CSS variables in your style sheets. When using CSS variables, the text color will not be automatically accessible. Use the background classes for this feature.
+
+<Source language="css" code={cssBackground} dark />
```

Comment:
A better solution to include code snippets can now be used:

<pre><code>```css dark
your snipped goes here
```</code></pre>